### DISCUSSION TITLE
sbt-release interactive release not working with sbtn

### DISCUSSION THREAD

**Discussion by jmvandebron** [sbt/sbt#7302] [Q&A]
Using SBT 1.9.0 and `"com.github.sbt" % "sbt-release" % "1.1.0"`

When I start sbt with `SBT_NATIVE_CLIENT=false`, then I can perform a `release` that will prompt me for the new version numbers (and other input), but when using `SBT_NATIVE_CLIENT=true`, I do not see all prompts, and pressing `Enter` does not continue the release.

Output when `SBT_NATIVE_CLIENT=false`:
```
...
sbt:root> release
[info] Starting release process off commit: 862ea891dc2f244d4106319a7815309dc5fc1222
[info] Checking remote [origin] ...
[info] Press enter to use the default value
Release version [0.2.1] :
Next version [0.2.2-SNAPSHOT] :
[success] Total time: 0 s, completed 18 Jun 2023, 14:22:51
...
```

With `SBT_NATIVE_CLIENT=true`, this becomes:
```
...
sbt:root> release
[info] Starting release process off commit: 2a71811d002272ca5755dec4ea4d5bf005407e46
[info] Checking remote [origin] ...
[info] Press enter to use the default value


^C
...
```

Is this a general problem with SBT when used in server + thin client mode, or is this a bug in the sbt-release plugin where it is not (yet) properly supporting the server + thin client setup?

**Comment by jmvandebron** [comment]
Could this be an issue with the call to `SimpleReader.readLine` (https://github.com/sbt/sbt-release/blob/5e907de2a1ea15ead14e297d70b944d62784508e/src/main/scala/ReleaseExtra.scala#LL299C42-L299C42)?

Neither the prompt shows up, nor is input accepted.

Perhaps the sbt server is not receiving all input as expected from the thin client?
(I read some questions where enter is being sent/received as 0x0A (LF) instead of 0x0D (CR), like https://github.com/sbt/sbt/issues/5900)

**Comment by SethTisue** [comment]
looks like https://github.com/sbt/sbt/issues/5900
